shared/src/msi-installer/WindowsInstaller.wixproj

```diff
@@ -18,7 +18,7 @@
     <SuppressPdbOutput>True</SuppressPdbOutput>
     <DefineSolutionProperties>false</DefineSolutionProperties>
     <OutputName>datadog-dotnet-apm-2.5.1-$(Platform)</OutputName> <!-- -The regex recognizes this line -->
-    <OutputName>$(OutputName)-profiler-beta</OutputName>
+    <OutputName>$(OutputName)</OutputName>
```

Comment:
nit: We can remove this line entirely
```suggestion
```

Reply:
Fixed in 6dc6dda249c28e100dbe28a2f5ad536fe359d625